janusgraph-backend-testutils/src/main/java/org/janusgraph/graphdb/GraphDatabaseConfigurationInstanceIdTest.java

```diff
@@ -19,85 +19,91 @@
         graph1.close();
         graph2.close();
     }
 
+    @Disabled("Not working anymore. The bug is tracked here: https://github.com/JanusGraph/janusgraph/issues/2696")
     @Test
     public void graphShouldNotOpenWithSameInstanceId() {
-        final Map<String, Object> map = new HashMap<>();
-        map.put(STORAGE_BACKEND.toStringWithoutRoot(), "inmemory");
-        map.put(UNIQUE_INSTANCE_ID.toStringWithoutRoot(), "not-unique");
+        final Map<String, Object> map = getStorageConfiguration();
+        map.put(UNIQUE_INSTANCE_ID.toStringWithoutRoot(), NON_UNIQUE_INSTANCE_ID);
         final MapConfiguration config = ConfigurationUtil.loadMapConfiguration(map);
         final StandardJanusGraph graph1 = new StandardJanusGraph(new GraphDatabaseConfigurationBuilder().build(new CommonsConfiguration(config)));
-
-        assertEquals(graph1.openManagement().getOpenInstances().size(), 1);
-        assertEquals(graph1.openManagement().getOpenInstances().toArray()[0], "not-unique");
+        assertEquals(1, graph1.openManagement().getOpenInstances().size());
+        assertEquals(NON_UNIQUE_CURRENT_INSTANCE_ID, graph1.openManagement().getOpenInstances().iterator().next());
         JanusGraphException janusGraphException = assertThrows(JanusGraphException.class, () -> {
             final StandardJanusGraph graph2 = new StandardJanusGraph(new GraphDatabaseConfigurationBuilder().build(new CommonsConfiguration(config)));
             graph1.close();
         });
-        assertEquals("A JanusGraph graph with the same instance id [not-unique] is already open. Might required forced shutdown.", janusGraphException.getMessage());
+        assertEquals("A JanusGraph graph with the same instance id ["+NON_UNIQUE_INSTANCE_ID+"] is already open. Might required forced shutdown.",
+            janusGraphException.getMessage());
     }
 
     @Test
     public void instanceIdShouldEqualHostname() throws UnknownHostException {
-        final Map<String, Object> map = new HashMap<>();
-        map.put(STORAGE_BACKEND.toStringWithoutRoot(), "inmemory");
+        final Map<String, Object> map = getStorageConfiguration();
         map.put(UNIQUE_INSTANCE_ID_HOSTNAME.toStringWithoutRoot(), true);
         final MapConfiguration config = ConfigurationUtil.loadMapConfiguration(map);
         final StandardJanusGraph graph = new StandardJanusGraph(new GraphDatabaseConfigurationBuilder().build(new CommonsConfiguration(config)));
-        assertEquals(graph.openManagement().getOpenInstances().size(), 1);
-        assertEquals(graph.openManagement().getOpenInstances().toArray()[0], Inet4Address.getLocalHost().getHostName());
-				graph.close();
+        assertEquals(1, graph.openManagement().getOpenInstances().size());
+        assertEquals(toCurrentInstance(Inet4Address.getLocalHost().getHostName()), graph.openManagement().getOpenInstances().iterator().next());
```

Comment:
This is failing locally for me. Do you know why?
```
Expected :liboxuans-MacBook-Pro.local(current)
Actual   :liboxuans-MacBook-Pro-local(current)
```

Reply:
Hmm interesting. It works for me locally. Looks it changed `.` to `-` for some reason

Reply:
Okay, I see the reason. Those characters are replaced here: `UniqueInstanceIdRetriever.computeUniqueInstanceId`

Reply:
@li-boxuan could you try now?

Reply:
Nice, they are working now